doc/release_notes.rst

```diff
@@ -90,6 +90,7 @@ controller_manager
 * The ``cpu_affinity`` parameter can now accept of types ``int`` or ``int_array`` to bind the process to a specific CPU core or multiple CPU cores. (`#1915 <https://github.com/ros-controls/ros2_control/pull/1915>`_).
 * The ``pal_statistics`` is now integrated into the controller_manager, so that the controllers, hardware components and the controller_manager can be easily introspected and monitored using the topics ``~/introspection_data/names`` and ``~/introspection_data/values`` (`#1918 <https://github.com/ros-controls/ros2_control/pull/1918>`_).
 * A python module ``test_utils`` was added to the ``controller_manager`` package to help with integration testing (`#1955 <https://github.com/ros-controls/ros2_control/pull/1955>`_).
+* The controller manager will use a monotonic clock for triggering read-update-write cycles, but when the ``use_sim_time`` parameter is set to true, it will use the ROS Clock for triggering. When monotonic clock is being used, All the hardware components will receive the monotonic time in their read and write method, instead the controllers will always receive the ROS time in their update method irrespective of the clock being used. (`#2046 <https://github.com/ros-controls/ros2_control/pull/2046>`_).
```

Comment:
```suggestion
* The controller manager will use a monotonic clock for triggering read-update-write cycles, but when the ``use_sim_time`` parameter is set to true, it will use the ROS Clock for triggering. When monotonic clock is being used, all the hardware components will receive the monotonic time in their read and write method, instead the controllers will always receive the ROS time in their update method irrespective of the clock being used. (`#2046 <https://github.com/ros-controls/ros2_control/pull/2046>`_).
```

Reply:
should we add this information to the permanent docs somewhere? Including the rationale behind this?
maybe [here](https://github.com/ros-controls/ros2_control/blob/master/controller_manager/doc/userdoc.rst#concepts) or split up in the [writing_new_hardware_component](https://github.com/ros-controls/ros2_control/blob/master/hardware_interface/doc/writing_new_hardware_component.rst) and [writing_new_controller](https://github.com/ros-controls/ros2_controllers/blob/master/doc/writing_new_controller.rst)

Reply:
Maybe we can add a section here about the clocks : 
https://github.com/ros-controls/ros2_control/blob/master/controller_manager/doc/userdoc.rst

Reply:
we already have the concepts section on this page, it could be below this section (the very end of the page)?

Reply:
Yes, I'm thinking at the end of the page

Reply:
(can be in a follow-up PR to avoid to block this being merged)

Reply:
Sure. Let's do that. Thank you